### ISSUE TITLE
Web UI at Director displays Origin capabilities but not namespace capabilities

### ISSUE THREAD

**Issue by jhiemstrawisc** [PelicanPlatform/pelican#1724]
When the Director receives an advertisement for an origin with multiple namespaces, it currently only displays the Origin capabilities, and not the capabilities of the namespaces themselves. Consider these two images:

![image](https://github.com/user-attachments/assets/a6f09219-d024-48ca-b2bb-6b9f97aada35)
![image](https://github.com/user-attachments/assets/7e83608c-b4a8-4a20-971f-305173bc9637)

The first shows the _namespace_ capabilities as they are configured at the origin. However, because the _Origin capabilities_ include writes and direct reads, the visual at the Director makes it seem like there's an error in advertising.

I think we need two fixes here -- first, to make it very clear at the Origin which capabilities are a property of the Origin, and which are properties of any namespaces. Second, we should be able to check this same information at the Director.

**Comment by CannonLock** [comment]
@jhiemstrawisc What is the point of the Origins capabilities? Where do those factor in if we are defining them on the namespace?

**Comment by jhiemstrawisc** [comment]
It gets down to how we manage the many-to-many relationship between origins and namespaces. Consider this situation:
```
.
└── Federation/
    ├── Origin1 # supports reads, not writes
    │   ├── /namespace1 # reads
    │   └── /namespace2 # reads, writes
    └── Origin2 # supports writes, not reads
        ├── /namespace2 # reads, writes
        └── /namespace3 # writes
```

Namespaces should be globally-defined (we're slowly working toward that), so any origin that serves a namespace should respect that namespace's capabilities. On the flip side, a particular origin might not want to support a capability, even if one of the namespaces it federates supports it.

When we're trying to pick which origin to send a request to, we do an AND operation on the origin's capabilities and the namespace capabilities -- that is, if you wanted to read from `/namespace2`, you'd be sent to origin1. If you wanted to write to `/namespace2`, you'd be sent to origin2.

**Comment by CannonLock** [comment]
@jhiemstrawisc Cool, thanks for the image, that helps a lot. I will make it happen :)

**Comment by CannonLock** [comment]
@jhiemstrawisc Is there any interaction when namespaces have a hierarchy?

For instance:
```
/major - Write
/major/minor - Read, Write
```

Or are they treated independently of each other?

**Comment by CannonLock** [comment]
Closed by #1737
